api/db/changelog/db.changelog-12.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+    <changeSet author="brianfreeman" id="changelog-12">
+
+        <createTable tableName="participant_cohort_status">
```

Comment:
As discussed offline: I think we're going to want another table, called "cohort_review" or something, that has metadata about a cohort review (e.g. when it started, what position you were in when reviewing participants, etc.) That will allow us to display what reviews are in progress, pick up where you left off, etc.

That could use (cohort ID, cdr version ID) as its PK, or have its own autogenerated ID with a unique index on the above. Either way it may make sense to have a FK from participant_cohort_status to that table.

Reply:
Agreed.

Reply:
OK, so that's coming in a follow-up? (And the endpoint to start a review?)

Reply:
Yes, I will create another story in jira for this work and complete it when i get back from vacation next week.